modules/ztp-image-based-upgrade-installing-lifecycle-agent-using-cli.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-lcao-using-cli_{context}"]
+= Installing the {lcao} by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the {lcao} on both the seed and target cluster.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the {rh-rhacm} Operator 2.9.2 or later.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
```

Comment:
```suggestion
.Create a namespace for the {lcao}.
[source,yaml]
----
apiVersion: v1
kind: Namespace
metadata:
  name: openshift-lifecycle-agent
  annotations:
    workload.openshift.io/allowed: management
----

..Create the Namespace CR:
[source,terminal]
----
$ oc create -f lcao-namespace.yaml
----

.Create an Operator group for the {lcao}.
[source,yaml]
----
apiVersion: operators.coreos.com/v1
kind: OperatorGroup
metadata:
  name: openshift-lifecycle-agent
  namespace: openshift-lifecycle-agent
spec:
  targetNamespaces:
  - openshift-lifecycle-agent
----

..Create the OperatorGroup CR:

[source,terminal]
----
$ oc create -f lcao-operatorgroup.yaml
----
```

Reply:
@jc-rh @rauhersu

Reply:
@amolnar-rh fwiw - above suggestion was taken from PTP operator official docs (which is a namespaced operator)

https://docs.openshift.com/container-platform/4.12/networking/using-ptp.html